### ISSUE TITLE
Consider marking adapters installed from github 

### ISSUE THREAD

**Issue by mcm1957** [ioBroker/ioBroker.admin#1640]
This is a low prio feature request / "nice to have" feature

As far as I know the information wheter an adapter is installed from repository or from external source like github is available at the target system. So it would be nice if at adapterview (Admin -> Adapter) a symbol near Senry symbol or some text or sysmbol near the version information could indicate that an adapter is not installed from central repository. As an alternative the versioninfo could be written itakic etc.

Background:
Currently ist not possible to check at adapter page whether an adapter is already at the released version. During development (at leasts I) install an adapter from github immidiatly before / after releasing it to latest to test it at my "production" in real life. So the version number is no safe indication for the source of the installation.

As this is a nice to have feature I  do not have any problems if the request will be rejected and closed

**Comment by Apollon77** [comment]
This is already implemented in Admin 5 ... ther eis a github icon if you installed it from GitHub ... there was an issue bit admin 6 should have that fixed too

**Comment by mcm1957** [comment]
Hmm ... if it should work, than this is a bug.
I do not see an icon :-(

**Comment by Apollon77** [comment]
Whats the value of "common.installedFrom" in the respective instance object (system.adapter.X.Y) ?

**Comment by mcm1957** [comment]
admin 5.3.8  - no icon 
    "installedFrom": "mcm1957/ioBroker.snmp#f48e9e55e98822f2f420e126b56e74fbf1ba6494",
![image](https://user-images.githubusercontent.com/28575778/184117592-3949ce0d-9f3a-43df-bb83-2be94ab84e3c.png)

admin 6.2.14 - no icon either
    "installedFrom": "mcm1957/ioBroker.snmp#f48e9e55e98822f2f420e126b56e74fbf1ba6494",
![image](https://user-images.githubusercontent.com/28575778/184118107-1587a5d2-d585-4915-938c-da75d4be6c2c.png)

I remember that I saw a "guthub cat" left of the version number once. (Not during this github install bute sometimes before.) It vanished when switching between pages. I think I did change repositories stable/lastest at this time. But this might be unrelated.
